src/App.js

```diff
@@ -1,9 +1,12 @@
-import './App.css';
+import ButtonPanel from './components/buttonPanel';
+import Display from './components/display';
+import './index.css';
 
 function App() {
   return (
     <div className="App">
-      <h1>Hello!</h1>
+      <Display result="0" />
+      <ButtonPanel />
     </div>
   );
 }
```

Comment:
## Changes on the `App` component
- [ ] This component should be inside the `src/components` directory. Let's move it to its rightful place ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/App.png)).
- [ ] While at it you might want to change it to an `arrow` functional component.
- [ ]  It's children components (`Display` & `ButtonPanel`) ought to be wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax as explained in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/). let's review this and make the necessary changes.
- below, I have shared what the final outcome for this component, in this second milestone should resemble. 👇 

```ruby
# src/components/App.js

import ButtonPanel from './components/buttonPanel';
import Display from './components/display';
import './index.css';

const App = () => (
  <>
    <Display result="0" />
    <ButtonPanel />
  </>
);

```